package.json

```
@@ -30,6 +30,7 @@
     "@typespec/versioning": "0.75.0",
     "@typespec/xml": "0.75.0",
     "azure-rest-api-specs-eng-tools": "file:eng/tools",
+    "cspell-version-pin": "file:eng/common/spelling",
```

```suggestion
    "@azure-tools/cspell": "file:eng/common/spelling",
```

Can you try this?  I think npm lets us rename like this.